content/docs/user-guide/what-is-dvc.md

```diff
@@ -0,0 +1,71 @@
+# What Is DVC?
+
+**Data Version Control** is a new type of data versioning, workflow and
+experiment management software, that builds upon [Git](https://git-scm.com/)
+(although it can work stand-alone). DVC reduces the gap between established
+engineering tool sets and data science needs, allowing users to take advantage
+of [new features](#core-features) while reusing existing skills and intuition.
+
+![](/img/reproducibility.png) _DVC codifies data and ML experiments_
+
+Data science experiment sharing and collaboration can be done through regular
+Git flows (commits, branching, pull requests, etc.), the same way it works for
+software engineers.
+
+[`dvc`](/doc/command-reference) is a command line tool, similar to `git`.
+
+## Core Principles
+
+- **Workflow**: Set of experiments and relationships among them. Corresponds to
+  the entire <abbr>project</abbr> and may contain several
+  [data pipelines](/doc/user-guide/basic-concepts#data-pipelines).
+
+- **Experiment**: An attempt at a data science task. Each one can be performed
```

Comment:
I wonder if there's a point in keeping these here in addition to the separate Basic Concepts guide. Are they basically the same? And if these are principles and not concepts, should they be explained differently? For example something like:

**Workflow**: DVC aims to provide a transparent experience in managing pipelines ... etc.

Something like

Reply:
yes, I think we can absorb those

Reply:
Done in eb5fbf99.